uw-frame-components/staticFeeds/features_template.json

```diff
@@ -16,16 +16,14 @@
       "startMonth": 7,
       "startDay": 11,
       "endYear": 2015,
-      "endMonth": 7,
+      "endMonth": 7, (0 based index)
```

Comment:
How safe is this approach to adding comments to `JSON`? Googling about this [feels bleak](http://stackoverflow.com/questions/244777/can-i-use-comments-inside-a-json-file)?